lib/utils/util.js

```diff
@@ -478,3 +478,256 @@ util.storeOneInCache = function (processedOutput, cache) {
+
+  return null;
+};
+
+/**
+ * @typedef {Object} AccountBackup
+ * @property {string} name Account Name
+ * @property {string} type Account Type
+ * @property {number} m M
+ * @property {number} n N
+ * @property {string[]} keys multisig keys (if m!=n and > 1)
+ * @property {number} receiveDepth number of receive addresses derived
+ * @property {number} changeDepth number of change addresses derived
+ *
+ */
+
+/**
+ * Restores an account and attaches it to the wallet
+ * using Backup
+ *
+ * @async
+ * @param {module:wallet.Wallet} wallet Wallet to attach account to
+ * @param {AccountBackup} accountBackup AccountBackup object to restore
+ * @returns {Promise<null>} null
+ */
+util.restoreAccount = async function (wallet, accountBackup) {
+  const {
+    name,
+    type,
+    m,
+    n,
+    keys,
+    receiveDepth,
+    changeDepth
+  } = accountBackup;
+
+  await wallet.ensureAccount({
+    name,
+    type,
+    m,
+    n,
+    keys
+  });
+
+  for (let i=0; i<(receiveDepth-1); i++) {
+      await wallet.createKey(name, 0);
+  }
+
+  for (let j=0; j<(changeDepth-1); j++) {
+      await wallet.createKey(name, 1);
+  }
+
+  return null;
+};
+
+/**
+ * Unzips a file and returns contents
+ * throws an error if something goes wrong
+ *
+ * @param {string} filePath local file path
+ * @returns {Buffer} unzipped contents
+ */
+util.unzipFile = function (filePath) {
+  const fs = require('fs');
+
+  if (!filePath) {
+    throw new Error('filepath is invalid (null|undefined)!');
+  }
+
+  if (!fs.existsSync(filePath)) {
+    throw new Error(`File: ${filePath} does not exist!`);
+  }
+
+  const buffer = fs.readFileSync(filePath);
+  return require('zlib').unzipSync(buffer);
+};
+
+/**
+ * Serialized an object into string with JSON.stringfy
+ * and zips the contents into a Buffer
+ *
+ * @param {Object} object to serializes to JSON and compress
+ * @returns {Buffer} compressed contents of object
+ */
+util.zipObject = function (object) {
+  return require('zlib').gzipSync(Buffer.from(JSON.stringify(object)));
+};
+
+/**
+* Return a timestamp with the format "d/m/yy h:MM:ss TT"
+* @type {Date}
+*/
+
+util.timeStamp = function timeStamp() {
+// Create a date object with the current time
+ const now = new Date();
+
+// Create an array with the current month, day and time
+ const date = [now.getDate(), now.getMonth() + 1, now.getFullYear()];
+
+// Create an array with the current hour, minute and second
+ const time = [now.getHours(), now.getMinutes(), now.getSeconds()];
+
+// Determine AM or PM suffix based on the hour
+ // const suffix = (time[0] < 12) ? 'AM' : 'PM';
+
+// Convert hour from military time
+ // time[0] = (time[0] < 12) ? time[0] : time[0] - 12;
+
+// If hour is 0, set it to 12
+ // time[0] = time[0] || 12;
+
+// If seconds and minutes are less than 10, add a zero
+ for (let i = 1; i < 3; i++) {
+   if (time[i] < 10) {
+     time[i] = '0' + time[i];
+   }
+ }
+
+// Return the formatted string
+ return date.join('/') + '_' + time.join(':');
```

Comment:
I think this function will not be necessary with some of the other requested changes (see other PR comments). The caller should be responsible for the filename IMO.

But fyi can use `new Date().toISOString()` here

Reply:
i have reverted to 

`new Date().toISOString() ` 

and removed this util function...